packages/orbit-components/src/InputField/index.tsx

```diff
@@ -143,154 +143,162 @@ const InputField = React.forwardRef<HTMLInputElement, Props>((props, ref) => {
 
   const shown = tooltipShown || tooltipShownHover;
   const fieldRef = React.useRef(null);
-  const Component = label ? "label" : "div";
 
   return (
-    <Component
+    <div
```

Comment:
I've changed the structure of the DOM to avoid to read the tooltip content by VO twice.